tests/yangre/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+find_program(PATH_TCLSH NAMES tclsh)
+if(NOT PATH_TCLSH)
+    message(WARNING "'tclsh' not found! The yangre will not be available.")
```

Comment:
You mean `yangre` tests will not be available, right?

Reply:
yes, fixed.